bag_of_words/code/bag_of_words.py

```diff
@@ -0,0 +1,36 @@
+# Read papers and convert them to bag-of-words representations.
+
+indir = "../data/raw/pubmed-txts-mf/"
+outdir = "../data/processed/"
+from sklearn.feature_extraction.text import CountVectorizer
+vectorizer = CountVectorizer()
+import os
+import numpy as np
+import pickle
+
+# Read papers into a list
+pmids = os.listdir(indir)
+pmids.sort()
+# pmids = pmids[:5]  # For debugging
+corpus = [''] * len(pmids)
+print("Reading corpus ...")
+for i in range(len(pmids)):
+    if i % 10000 == 0:
+        print(str(i) + " docs processed ...")
+    for field in os.listdir(indir + pmids[i] + '/'):
+        with open(indir + pmids[i] + '/' + field, 'r') as fin:
+            text = fin.read().replace('\n', ' ')
+            corpus[i] += text
+print('')

```

Comment:
I recommend using python's `logger` module, which you can simply use in files like this by using `logging.getLogger()`